### ISSUE TITLE
Cookies are not send with the request

### ISSUE THREAD

**Issue by mariosarbinov** [m-inan/react-upload-gallery#36]
XMLHttpRequest  doesn't send the cookies by default. So it will be great if you set `xhr.withCredentials = true;` by default or extract property that will set it.

**Comment by m-inan** [comment]
For a quick solution you can use the [customRequest](https://github.com/m-inan/react-upload-gallery#customrequest) props.

**Comment by mariosarbinov** [comment]
> 

Yes, I saw that option but it's a bit overkill so that's why I am wondering do you plan to handle this use case in the library soon?

**Comment by m-inan** [comment]
You can use it as a prop in version [0.4.1](https://github.com/m-inan/react-upload-gallery/releases/tag/v0.4.1). I've been thinking about rewriting the project for a long time, so I'm postponing such updates.

**Comment by mariosarbinov** [comment]
Thank you very much for the update. It works perfect! You saved my day! I hope you will put some notification in this project if you rewrite it :) Good luck!
